src/components/Panels/TimingDiagramPanel/TimingDiagramPanel.vue

```diff
@@ -57,15 +71,12 @@ interface TimingDiagramButton {
     click: string
 }
 
-interface PlotArea {
-    resize: () => void
-    [key: string]: () => void
-}
-
-const plotArea: PlotArea = _plotArea
+const plotArea = reactive(_plotArea)
+const timingDiagramStore = useTimingDiagramStore()
 const buttons = ref<TimingDiagramButton[]>(buttonsJSON)
 const plotRef = ref<HTMLElement | null>(null)
-const cycleUnits = ref(1000)
+const utilization = computed(() => (Math.round((plotArea.unitUsed * 10000) / plotArea.cycleUnit) / 100))
+const recommendedUnit = computed(() => (Math.max(20, Math.round(plotArea.unitUsed * 3))))
```

Comment:
Let's start using `computed` suffix for the naming.
```suggestion
const utilizationComputed = computed(() => (Math.round((plotArea.unitUsed * 10000) / plotArea.cycleUnit) / 100))
const recommendedUnitComputed = computed(() => (Math.max(20, Math.round(plotArea.unitUsed * 3))))
```